Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu/postgres:12-20.04_beta
+
+RUN echo 'APT::Install-Recommends "0";\nAPT::Install-Suggests "0";' > /etc/apt/apt.conf.d/01norecommend \
+    && apt-get update -y \
+    # Install Patroni and its dependencies.
+    && apt-cache depends patroni | sed -n -e 's/.* Depends: \(python3-.\+\)$/\1/p' \
+            | grep -Ev '^python3-(sphinx|etcd|consul|kazoo|kubernetes)' \
```

Comment:
@marceloneppel , why do you need to avoid this dependencies?

Reply:
Exactly, @paulomach. These dependencies are not used.